src/theme/index.hbs

```diff
@@ -144,8 +144,15 @@
         ga('create', '{{google_analytics}}', 'auto');
         ga('send', 'pageview');
         </script>
-      {{/if}}
-
+        {{/if}}
+
+        {{#if ace}}
```

Comment:
maybe we can use `editable-playpens`here and leave ace as an implementation detail